04_numeronym/undrewb/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	strs := make([]string, len(vals))
+	for i, s := range vals {
+		strs[i] = numeronym(s)
+	}
+	return strs
+}
+
+func numeronym(s string) string {
+	outs := s
```

Comment:
@undrewb , neat implementation.  You can possibly do without another variable outs here though.
Just return the formatted string when you can build a numeronym or return the string s otherwise.

Reply:
fixed